Core/PixelEvent.swift

```diff
@@ -1326,7 +1325,6 @@ extension Pixel.Event {
         case .cachedTabPreviewRemovalError: return "m_d_tpre"
             
         case .missingDownloadedFile: return "m_d_missing_downloaded_file"
-        case .unhandledDownload: return "m_d_unhandled_download"
```

Comment:
Looking at the code path and confirming in Kibana this pixel is/was not being fired.